reflex/utils/processes.py

```diff
@@ -320,7 +320,8 @@ def stream_logs(
 
     # Windows uvicorn bug
     # https://github.com/reflex-dev/reflex/issues/2335
-    accepted_return_codes = [0, -2, 15] if constants.IS_WINDOWS else [0, -2]
+    # 130 is the exit code that react router returns when it is interrupted by a signal.
+    accepted_return_codes = [0, -2, 15, 130] if constants.IS_WINDOWS else [0, -2, 130]
```

Comment:
```suggestion
    accepted_return_codes = [0, -2, 15] if constants.IS_WINDOWS else [0, -2, 130]
```

seems 130 isn't strictly needed on windows, but probably doesn't hurt to have it there

Reply:
it's ok then